script/rubyprof

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env bash
-
-export BENCHMARK=1
-
-TEST_SCRIPT="Jekyll::Commands::Build.process({'source' => 'site'})"
-
-RUBY=$(cat <<RUBY
-  require 'ruby-prof'
-  result = RubyProf.profile{ ${TEST_SCRIPT} }
-  printer = RubyProf::CallTreePrinter.new(result)
-  filename = "tmp/ruby_prof_#{rand 10000}"
-  puts "Writing profile to #{filename}"
-  file = File.open(filename, "w")
-  printer.print(file, {})
-  file.close
-RUBY
-)
-
-bundle exec ruby -r ./lib/jekyll -e "${RUBY}"
```

Comment:
Again, this pull isn't finished, wait! It will be put back in *this* pull, it was removed in this pull and will be rebuilt as part of the Jekyll command in this pull.